### ISSUE TITLE
https://github.com/terrychenism/ResNeXt link is broken

### ISSUE THREAD

**Issue by mrgloom** [beniz/deepdetect#358]
https://github.com/terrychenism/ResNeXt link is broken and I can't see ResNext model in this repo https://github.com/beniz/deepdetect/tree/master/templates/caffe

**Comment by beniz** [comment]
Right, good catch! The models are available from here: https://deepdetect.com/models/resnext/
We'll update the links.
